### PR TITLE
aspelldict PG: New PortGroup

### PR DESCRIPTION
**Pull request by judaew**
#### Description

The proposed PG is similar to hunspelldict-1.0. The main goal of the PG is to avoid repetitive ports configuration for aspell dictionaries. This will make aspell dictionaries maintenance much easier.

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 12.4 21F79 x86_64
Command Line Tools 13.4.0.0.1.1651278267

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Review by mascguy — APPROVED** [review]
Wow, this certainly simplifies the various dictionaries. Love it!

File: _resources/port1.0/group/aspelldict-1.0.tcl
Comment (reneeotten) [inline _resources/port1.0/group/aspelldict-1.0.tcl:26]: this is already the default and doesn't need to be set anymore

**Review by reneeotten — COMMENTED** [review]
thanks @judaew - one tiny comment but this looks very beneficial! It would be helpful though to give some time for people that are asked for a review to actual respond before merging things.

File: _resources/port1.0/group/aspelldict-1.0.tcl
Comment (judaew) [reply _resources/port1.0/group/aspelldict-1.0.tcl:26]: Thank you, I forgot that platforms are already by default

**Comment by judaew** [comment]
> thanks @judaew - one tiny comment but this looks very beneficial! It would be helpful though to give some time for people that are asked for a review to actual respond before merging things.

Sorry, but I wasn't the one who merged these changes, maybe Christopher was a bit quickdraw. I expected to merge this PR after your reply.

**Comment by mascguy** [comment]
> > thanks @judaew - one tiny comment but this looks very beneficial! It would be helpful though to give some time for people that are asked for a review to actual respond before merging things.
> 
> Sorry, but I wasn't the one who merged these changes, maybe Christopher was a bit quickdraw. I expected to merge this PR after your reply.

Yep, I chose to merge, to keep our PR queue moving. And I suspected that your situation had recently changed Renee, which you've now confirmed.

**Comment by judaew** [comment]
Anyway, thanks Renee and Christopher for helping make this PR better.

**Pull request merged**
